### ISSUE TITLE
12,400+ of "unknown files" are ".xz" - Mark them appropriately

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3333]
12,400+ of "unknown files" (9.990% of unknown files) have extension `.xz`

---

Fixes:
- [x] ~~**Now**: Mark these as compressed files instead of unknown.~~
- [ ] **By end of year**: Alternately, re-crawl these files and save the file list as metadata
  - Depends on #3088

---

Filed under: tech debt

![harvard_dataverse](https://cloud.githubusercontent.com/assets/1036060/18289849/22b3fbfa-744f-11e6-95ba-7a8a08d35070.png)

---

``` json
  "data": {
    "number_unique_extensions": 1454,
    "file_extension_counts": [
      {
        "extension": "",
        "count": 36657,
        "total_count": 124357,
        "percent_string": "29.477%"
      },
      {
        "extension": ".xz",
        "count": 12423,
        "total_count": 124357,
        "percent_string": "9.990%"
      },
  (etc)
```

**Comment by mheppler** [comment]
Related to #2202

**Comment by pdurbin** [comment]
Closing in favor of #2202
